.travis.yml

```diff
@@ -28,6 +28,8 @@ script:
   - |
     if [[ "${TRAVIS_JDK_VERSION}" == "oraclejdk8" && ( "${TRAVIS_BRANCH}" == "master" || "${TRAVIS_EVENT_TYPE}" == "pull_request" || "${TRAVIS_EVENT_TYPE}" == "cron" ) ]]; then
       ./gradlew -PgeckoDriverPlatform=linux64 -PgeckoDriverVersion=0.20.0 build testAll
+    elif [[ "${TRAVIS_JDK_VERSION}" == "oraclejdk9" ]]; then
```

Comment:
Rather than a nested `elif`, could this work near the top of the script, exporting `JDK_JAVA_OPTIONS`?

Reply:
Yup, fixed :)